test/index.spec.js

```diff
@@ -61,11 +61,10 @@ describe('what is open', () => {
     expect(listitems.length).toBeGreaterThan(1);
     await page.click('#awesomplete_list_1 li');
 
-    await page.type('#location-query', 'San Diego');
```

Comment:
It looks like this test was searching a compound county string ("San BenitoSan Diego"). With my change, this search no longer works. I removed this typing so the test just searches "San Benito".

![test](https://user-images.githubusercontent.com/213565/101957044-86b41400-3bb5-11eb-96e2-85ab29bde2c4.jpg)

Reply:
Hmmm. This test _looks_ like it's passing locally for me but is not passing here. I will take closer look soon.